core/templates/dev/head/pages/exploration_player/InputResponsePairDirective.js

```diff
@@ -67,6 +72,19 @@ oppia.directive('inputResponsePair', [
             }
             return shortAnswerHtml;
           };
+
+          $scope.getFeedbackAudioHighlightClass = function() {
+            if (!$scope.last) {
+              return '';
+            }
+            if (AudioTranslationManagerService
+              .getCurrentComponentName() ===
+              COMPONENT_NAME_FEEDBACK &&
+              (AudioPlayerService.isPlaying() ||
+              AutogeneratedAudioPlayerService.isPlaying())) {
+              return 'conversation-skin-audio-highlight';
```

Comment:
Make this string an oppia.constant; put a comment above the CSS class definition to say its name should match this constant.

Reply:
done